io-uring-test/src/tests/net.rs

```diff
@@ -1436,6 +1436,135 @@ pub fn test_udp_recvmsg_multishot<S: squeue::EntryMarker, C: cqueue::EntryMarker
 
     Ok(())
 }
+pub fn test_udp_recvmsg_multishot_trunc<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
+    ring: &mut IoUring<S, C>,
+    test: &Test,
+) -> anyhow::Result<()> {
+    require!(
+        test;
+        test.probe.is_supported(opcode::RecvMsgMulti::CODE);
+        test.probe.is_supported(opcode::ProvideBuffers::CODE);
+        test.probe.is_supported(opcode::SendMsg::CODE);
+        test.check_kernel_version("6.6.0");
+    );
+
+    println!("test udp_recvmsg_multishot_trunc");
+
+    let server_socket: socket2::Socket = std::net::UdpSocket::bind("127.0.0.1:0").unwrap().into();
+    let server_addr = server_socket.local_addr()?;
+
+    const BUF_GROUP: u16 = 33;
+    const DATA: &[u8] = b"testfooo for me";
+    let mut buf1 = [0u8; 20];
+    let mut buf2 = [0u8; 20 + DATA.len()];
```

Comment:
The `20` here is a bit magic, it would be good to note where it is coming from (is it `size_of(io_uring_recvmsg_out)`?)